### ISSUE TITLE
verified logo not displaying until panel is opened for Twitch publishers

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#5848]
## Description 
Follow up to https://github.com/brave/brave-browser/issues/5549

When navigating to a verified or connected Twitch publisher page, the verified icon doesn't display on the BAT logo until you open the panel. If you navigate to another page and come back, the verified logo isn't displayed again (until you open the panel).

## Steps to Reproduce

   1. Install 0.71.51 Nightly
   2. Enable Rewards
   3. Navigate to a verified Twitch publisher (if using staging, you can go to twitch.tv/laurenwags) or connected Twitch publisher (if using staging, you can go to twitch.tv/ilikeblueturtles)

## Actual result:
Verified logo isn't displayed on the BAT icon until you open the panel.
![Screen Shot 2019-08-30 at 3 34 30 PM](https://user-images.githubusercontent.com/28145373/64047094-cdeed080-cb3b-11e9-93cf-d23bee45d1df.png)

![Screen Shot 2019-08-30 at 3 34 47 PM](https://user-images.githubusercontent.com/28145373/64047099-d0e9c100-cb3b-11e9-9e0c-f148b63d5f01.png)

## Expected result:
Verified icon displayed on BAT logo without having to open panel first.
![Screen Shot 2019-08-30 at 3 34 56 PM](https://user-images.githubusercontent.com/28145373/64047088-c7f8ef80-cb3b-11e9-85dd-fe02b535ed04.png)

## Reproduces how often: 
easily

## Brave version (brave://version info)

Brave | 0.71.51 Chromium: 76.0.3809.132 (Official Build) nightly (64-bit)
-- | --
Revision | fd1acc410994a7a68ac25bc77513d443f3130860-refs/branch-heads/3809@{#1035}
OS | Mac OS X

## Version/Channel Information:

- Can you reproduce this issue with the current release? n/a 
- Can you reproduce this issue with the beta channel? n/a
- Can you reproduce this issue with the dev channel? n/a
- Can you reproduce this issue with the nightly channel? yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? n/a
- Does the issue resolve itself when disabling Brave Rewards? yes
- Is the issue reproducible on the latest version of Chrome? n/a

## Miscellaneous Information:
This doesn't happen for regular sites, YouTube, Vimeo, GitHub, or Reddit publishers.

**Comment by GeetaSarvadnya** [comment]
The issue is Reproducible in ` BraveNightly 0.72.3` staging environment - Until I open BR panel verified icon is not displayed on BAT logo in URL bar

![image](https://user-images.githubusercontent.com/38657976/64453314-dfb91080-d105-11e9-813e-d1488e0d0b2c.png)

**Comment by Miyayes** [comment]
@LaurenWags @GeetaSarvadnya Is this still reproducible in recent versions of Brave as of July 2022?

**Comment by GeetaSarvadnya** [comment]
The issue is fixed and working fine on `Windows 10 x64 - 1.42.86`. The verified logo is displayded as soon as a twitch publisher is visited

```
Brave | 1.42.86 Chromium: 104.0.5112.81 (Official Build) (64-bit)
-- | --
Revision | 5b7b76419d50f583022568b6764b630f6ddc9208-refs/branch-heads/5112@{#1309}
OS | Windows 10 Version 21H2 (Build 19044.1826)

```

![Twitch pubs](https://user-images.githubusercontent.com/38657976/182632721-06900014-64e2-42f3-9827-4b175d73ac34.gif)

**Comment by LaurenWags** [comment]
Thanks @GeetaSarvadnya! Just added some labels and a milestone to this one 👍🏻
